### DISCUSSION TITLE
Reverse Alias Replacement Feature

### DISCUSSION THREAD

**Discussion by jumpingbrowntrout** [simple-login/app#545] [Q&A]
Is anyone able to advise technically how this feature works?

The description of the feature is confusing however there is a good explanation of what it is supposed to do on this reddit
https://www.reddit.com/r/Simplelogin/comments/k1c0o5/reverse_alias_replacement_confusion/

From what I understand it's supposed to replace the original sender address (alias) within the reply with the original sender address. So is simplelogin doing this leg work manipulation after the mail is sent or is the client supposed to do it?

I am using protonmail and it doesn't seem to work. I would like to understand how it's supposed to work so I can see if I can request a feature via protonmail. 

I tested in Thunderbird and it seems to remove the alias address but it doesn't replace it with the original.

I inspected the headers and the source and I cannot see any reference to the original sender address which is why I am confused how it is achieved by the client.

**Comment by nguyenkims** [comment]
This feature is indeed experimental and we haven't tried it on Thunderbird yet. Its logic is pretty simple: SimpleLogin looks for the reverse-alias and replaces it with the sender address in the email content when you reply to a forwarded email. This logic doesn't handle the case where the email client applies an additional transformation to the reply before the reply is sent. More info can be found on https://github.com/simple-login/app/issues/418

**Reply by jumpingbrowntrout** [reply]
Thank you I shall reply there moving folder!
